test/k8s/deployment.test.ts

```diff
@@ -73,6 +74,39 @@ suite('K8s/deployment', () => {
         sandbox.restore();
     });
 
+    suite('DeploymentConfigNodeContributor', () => {
+        let kubectl;
+        const parent = <ClusterExplorerV1.ClusterExplorerNode>{
+            metadata: undefined,
+            name: "comp1-app",
+            namespace: null,
+            nodeType: "resource",
+            resourceKind: {
+                abbreviation: "dc",
+                description: "",
+                displayName: "DeploymentConfigs",
+                label: "DeploymentConfigs",
+                manifestKind: "DeploymentConfig",
+                pluralDisplayName: "DeploymentConfigs"
+            }
+        };
+
+        setup(async () => {
+            kubectl = await extension.kubectl.v1;
+        });
+
+        test('should able to get the children node of deployment Config', async () => {
+            if (kubectl.available) {
+                sandbox.stub(kubectl.api, 'invokeCommand').resolves({code: 0,
```

Comment:
Would be good to test with actual output here, so getChildren would return at least two nodes.